metrology/reporter/statsd.py

```diff
@@ -0,0 +1,112 @@
+import re
+import socket
+import sys
+
+from metrology.instruments import *  # noqa
+from metrology.reporter.base import Reporter
+
+
+class StatsDReporter(Reporter):
+    """
+    A statsd reporter that send metrics to statsd daemon ::
+
+      reporter = StatsDReporter('statsd.local', 2003)
```

Comment:
There are no default values for this reporter, host and port are required arguments. Host doesn't matter here, but I changed port, to avoid misunderstandings. Thanks!